great-mystery-great-whore/original/113-scotland-priests.adoc

```diff
@@ -21,25 +21,33 @@ which Christ and the apostles did not.
 George Weare, John Hart, William Grey, William Lowry, William Mitchell,
```

Comment:
I'm not sure if these names should be in this continuation blurb? Or as SSS? I left them in the continuation blurb just in case you had made them that way.